app/views/devise/mailer/password_change.text.erb

```diff
@@ -0,0 +1,4 @@
+Hello <%= @resource.email %>!
+
+We're contacting you to notify you that your password has been changed.
+
```

Comment:
```suggestion
```

Reply:
Nope, just apply the diff I commented on. The indentation in text emails causes it to be indented when rendered.